core/runtime.rs

```diff
@@ -788,6 +795,18 @@ impl JsRuntime {
       self.check_promise_exceptions()?;
     }
 
+    // Event loop middlewares
```

Comment:
Updated where middlewares are called. For NAPI, we need it after checking for promise exceptions.

https://github.com/denoland/deno/pull/13633/files#diff-5b779288098bb6449725245ce7fdb1ec3190563b5dd255e747cfc35e639407aeR854